code/modules/mob/living/simple_animal/hostile/changeling.dm

```diff
@@ -0,0 +1,136 @@
+/mob/living/simple_animal/hostile/true_changeling
+	name = "shambling horror"
+	desc = "A monstrous creature, made of twisted flesh and bone."
+	speak_emote = list("gibbers")
+	icon = 'icons/mob/animal.dmi'
+	icon_state = "horror"
+	icon_living = "horror"
+	icon_dead = "horror_dead"
+	stop_automated_movement = 1
+	universal_speak =1
+	universal_understand = 1
+
+	mob_swap_flags = HUMAN|SIMPLE_ANIMAL|SLIME|MONKEY
+	mob_push_flags = ALLMOBS
+
+	response_help  = "pets"
+	response_disarm = "shoves"
+	response_harm   = "harmlessly punches"
+	maxHealth = 750
+	health = 750
+	harm_intent_damage = 0
+	melee_damage_lower = 30
+	melee_damage_upper = 30
+	mob_size = 25
+	environment_smash = 2
+	attacktext = "mangled"
+	attack_sound = 'sound/weapons/bloodyslice.ogg'
+
+	see_in_dark = 8
+	see_invisible = SEE_INVISIBLE_NOLIGHTING
+
+	minbodytemp = 0
+	maxbodytemp = 350
+	min_oxy = 0
+	max_co2 = 0
+	max_tox = 0
+	
+	var/is_devouring = FALSE
+	
+/mob/living/simple_animal/hostile/true_changeling/Initialize()
+	. = ..()
+	if(prob(25))
+		icon_state = "horror_alt"
+		icon_living = "horror_alt"
+	
+/mob/living/simple_animal/hostile/true_changeling/Life()
+	..()
+	adjustBruteLoss(-10) //it will slowly heal brute damage, making fire/laser a stronger option
+	
+/mob/living/simple_animal/hostile/true_changeling/mind_initialize()
+	..()
+	mind.assigned_role = "Changeling"
+	
+/mob/living/simple_animal/hostile/true_changeling/death(gibbed)
+	..()
+	if(!gibbed)
+		visible_message("<b>[src]</b> lets out a waning scream as it falls, twitching, to the floor!")
+		playsound(loc, 'sound/effects/creepyshriek.ogg', 30, 1)
+		gibs(src.loc)
+		qdel(src)
+		return
+
+/mob/living/simple_animal/hostile/true_changeling/verb/ling_devour(mob/living/target as mob in oview())
+	set category = "Changeling"
+	set name = "Devour"
+	set desc = "Devours a creature, destroying its body and regenerating health."
+
+	if(!Adjacent(target))
+		return
+		
+	if(target.isSynthetic())
+		return
+		
+	var/mob/living/simple_animal/hostile/true_changeling/M = usr
```

Comment:
woups, I deleted a comment.

This var is unnecessary. All references to `M` can be replaced with `src` or deleted.